addons/mail/static/src/web/chatter.xml

```diff
@@ -153,7 +153,7 @@
 </t>
 
 <t t-name="mail.Chatter.attachFiles" owl="1">
-    <button class="o-mail-Chatter-attachFiles btn btn-link text-action px-1 d-flex align-items-center" aria-label="Attach files" t-att-class="{ 'my-2': !props.compactHeight }" t-on-click="onClickAddAttachments">
+    <button class="o-mail-Chatter-attachFiles btn btn-link text-action px-1 d-flex align-items-center" aria-label="Attach files" t-att-class="{ 'my-2': !props.compactHeight }" t-att-disabled="attachments.length === 0" t-on-click="onClickAddAttachments">
```

Comment:
No attachments is no the correct condition to disable the button.
The commit is about disabling read-only documents. A chatter can have no attachments and the button should work to upload at least 1 attachment.

Reply:
Updated the condition